test/models/billing_profile_test.rb

```diff
@@ -102,4 +102,13 @@ def test_create_default_for_user_creates_a_billing_profile
     assert_equal(billing_profile.country_code, user_without_billing_profile.country_code)
     assert_equal(billing_profile.name, user_without_billing_profile.display_name)
   end
+
+  def test_billing_address_is_mirrored_to_linked_invoice
+    # Invoice which is linked with @billing_profile
+    invoice = invoices(:payable)
+    @billing_profile.update(name: "New Company Ltd")
```

Comment:
- Single quotes.
- Can we extract this literal into variable `billing_profile_name` (this name sounds weird to me, in fact, but this is a design problem, and out of this PR's context) and reuse it?